newdle/api.py

```diff
@@ -355,6 +357,21 @@ def update_participant(args, code, participant_code):
     for key, value in args.items():
         setattr(participant, key, value)
     db.session.commit()
+    if participant.newdle.notify:
+        notify_newdle_creator(
+            participant.newdle,
+            participant,
+            f'{participant.name} joined to {participant.newdle.title}',
+            'joined_email.txt',
+            'joined_email.html',
+            lambda p: {
+                'participant': participant.name,
+                'title': participant.newdle.title,
+                'answer_link': url_for(
+                    'newdle', code=participant.newdle.code, participant_code=participant.code, _external=True
+                ),
+            },
```

Comment:
Unlike `notify_newdle_participants` this one is always about a single participant, so we don't need the lambda and can instead pass the context directly.